ichnaea/taskapp/task.py

```diff
@@ -13,12 +13,6 @@
 class BaseTask(Task):
     """A base task giving access to various outside connections."""
 
-    # BBB: Celery 4 non-underscore attributes
-    abstract = True
-    acks_late = False
-    ignore_result = True
-    max_retries = 3
```

Comment:
What happened to these? Are they accounted for somewhere else?

Reply:
Yep, these are no longer needed, set in configuration, or already the default.

[The Task base class no longer automatically registers tasks](https://docs.celeryproject.org/en/latest/history/whatsnew-4.0.html#the-task-base-class-no-longer-automatically-register-tasks). Instead, they can be registered manually or with the ``@celery_app.task`` decorator. This change also means that the ``abstract`` attribute of the task no longer has any effect.

The default for [tasks_acks_late](https://docs.celeryproject.org/en/latest/userguide/configuration.html#task-acks-late) is Disabled, so this ``acks_late = False`` is redundant.

The default for [task_ignore_result](https://docs.celeryproject.org/en/latest/userguide/configuration.html#task-ignore-result) is Disabled, but we have ``task_ignore_result = True`` in ``ichnaea.taskapp.settings``, so it is redundant here.

The default for [max_retries](https://github.com/celery/celery/blob/8cf8fae70630954cff3448485588bbe2a77ff3ab/celery/app/task.py#L178-L180) is ``3``, but it is undocumented and you have to peek at the code.